### PR TITLE
Setup: Read ILIAS version from constant

### PR DESCRIPTION
**Pull request by mjansenDatabay**
This PR replaces the statically defined version number in src/Setup with the ILIAS_VERSION_NUMERIC constant.
This constant is defined in the include/inc.ilias_version.php file and bumped by the release manager.

Of course this introduces a dependency on this global constant.

### CODE REVIEW

File: src/Setup/Condition/PHPExtensionLoadedCondition.php [hunk 1]
```diff
@@ -10,12 +10,14 @@ class PHPExtensionLoadedCondition extends ExternalConditionObjective
...
     {
+        $ilias_version = ILIAS_VERSION_NUMERIC;
+
         return parent::__construct(
             "PHP extension \"$which\" loaded",
             function (Setup\Environment $env) use ($which) : bool {
                 return in_array($which, get_loaded_extensions());
             },
-            "ILIAS 7 requires the PHP extension $which."
+            "ILIAS $ilias_version ILIAS_VERSION_NUMERIC . " . " requires the PHP extension $which."
```
Comment (PurHur) [inline src/Setup/Condition/PHPExtensionLoadedCondition.php]: Could it be that `ILIAS_VERSION_NUMERIC` was slipped in the string here?

**Review by PurHur — APPROVED** [review]

File: src/Setup/Condition/PHPExtensionLoadedCondition.php (hunk 1 above)
Comment (mjansenDatabay) [reply src/Setup/Condition/PHPExtensionLoadedCondition.php]: @PurHur What do you mean :)?

Comment (PurHur) [reply src/Setup/Condition/PHPExtensionLoadedCondition.php]: Sorry i forgot to click submit.

**Comment by klees** [comment]
Thanks!

**Pull request merged**